src/tools/interop/idt/Dockerfile

```
@@ -0,0 +1,58 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+FROM debian:bookworm
+
+# deps
+RUN apt-get update
+RUN apt-get install -y apt-utils git wget
```

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments

```suggestion
RUN apt-get install -y apt-utils git wget
```
Check out the [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments) doc.
```suggestion
RUN apt-get install -y apt-utils git wget
```

```bash
RUN apt-get update && apt-get install -y \
  bzr \
  cvs \
  git \
  mercurial \
  subversion \
  && rm -rf /var/lib/apt/lists/*
```

---

Thanks, hopefully my update is acceptable!